reporters_db/data/regexes.json

```diff
@@ -78,6 +78,8 @@
         "with_alpha_suffix#": "Volume number that may have 'A' appended, like '1A'",
         "with_digit_suffix": "(?P<volume>\\d{1,4}(?:-\\d+)?)",
         "with_digit_suffix#": "Volume number that may have digits appended, like '123-4'",
+        "with_letter": "(?P<volume>\\d+[a-zA-Z])",
+        "with_letter#": "Volume number followed by a letter",
```

Comment:
I'm a little worried about typos getting sucked into these by allowing any Letter.

Reply:
I updated the regex with the letters i found in the citations